website/addons/osfstorage/routes.py

```diff
@@ -112,6 +112,24 @@
             views.osfstorage_copy_hook,
             json_renderer,
         ),
+
+        Rule(
+            [
+                '/project/<pid>/osfstorage/<fid>/tags/<tag>/',
```

Comment:
Standard confirmation question: Have the two new routes introduced in this PR been approved by routing tsars BrianG and/or Jeff?